### PR TITLE
Add support for TECH VNTH-T2 Smart Radiator Valve (Tuya thermostat)

### PR DESCRIPTION
**Pull request by christophe-r**
Add support for Smart Radiator Valve (Tuya thermostat) :
Brand: **TECH**
Model: **VNTH-T2**
Fingerprint: **_TZE204_r7brscr6**

The image file has been added to zigbee2mqtt.io:
https://github.com/Koenkk/zigbee2mqtt.io/blob/master/public/images/devices/VNTH-T2.png

### CODE REVIEW

**Comment by Koenkk** [comment]
Please use `datapoints` instead, [example Tuya thermostat](https://github.com/Koenkk/zigbee-herdsman-converters/blob/f7110985024ac51bb1523ac389de843abf186f95/src/devices/tuya.ts#L4393)

**Comment by christophe-r** [comment]
@Koenkk Thanks for your review.

It's done.

Indeed, it's much simpler :)

File: src/devices/tech.ts [hunk 1]
```diff
@@ -0,0 +1,103 @@
+import {ota} from '../index';
```
Comment (Koenkk) [inline src/devices/tech.ts:1]: Please also add this file to https://github.com/Koenkk/zigbee-herdsman-converters/blob/master/src/devices/index.ts

Comment (Koenkk) [inline src/devices/tech.ts:1]:
```suggestion
import * as ota from '../lib/ota';
```

**Comment by christophe-r** [comment]
@Koenkk All done

**Pull request merged**